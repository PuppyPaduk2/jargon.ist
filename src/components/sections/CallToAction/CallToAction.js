// @flow

import React from 'react';

import {} from '../../ui/LinkButton';

const CallToAction = () => (
  <div className="u-text-center">
    <h1 className="u-color-primary u-clear-gap">
      <span className="u-hidden@sm-down">sen de </span>katkıda bulun
    </h1>

    <p className="u-color-comet u-gap-top-xsmall u-gap-bottom ">
      Yeni bir jargon eklemek için sadece bir markdown dosyası hazırlaman yeterli.
    </p>

    <a
      href={`${process.env.GATSBY_REPO_URL || ''}/blob/master/CONTRIBUTING.md`}
      target="_blank"
      rel="noopener noreferrer"
      className="c-button c-button--primary"
    >
      Yeni Jargon Ekle
    </a>
  </div>
);

export default CallToAction;
