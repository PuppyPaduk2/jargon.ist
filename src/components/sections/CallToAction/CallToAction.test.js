// @flow

import React from 'react';
import { shallow } from 'enzyme';

import CallToAction from './CallToAction';

it('renders correctly', () => {
  const wrapper = shallow(<CallToAction />);

  expect(wrapper).toMatchSnapshot();
});

it('contains a button to contribution document', () => {
  const wrapper = shallow(<CallToAction />);

  expect(wrapper.find('a').prop('href')).toContain('CONTRIBUTING.md');
});
