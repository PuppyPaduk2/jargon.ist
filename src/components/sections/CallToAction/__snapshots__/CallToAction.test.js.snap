// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="u-text-center"
>
  <h1
    className="u-color-primary u-clear-gap"
  >
    <span
      className="u-hidden@sm-down"
    >
      sen de 
    </span>
    katkıda bulun
  </h1>
  <p
    className="u-color-comet u-gap-top-xsmall u-gap-bottom "
  >
    Yeni bir jargon eklemek için sadece bir markdown dosyası hazırlaman yeterli.
  </p>
  <a
    className="c-button c-button--primary"
    href="/blob/master/CONTRIBUTING.md"
    rel="noopener noreferrer"
    target="_blank"
  >
    Yeni Jargon Ekle
  </a>
</div>
`;
